app/models/chargeback.rb

```diff
@@ -170,4 +170,8 @@ def self.get_report_time_range(options, interval, tz)
   def self.report_cb_model(model)
     model.gsub(/^Chargeback/, "")
   end
+
+  def self.chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
I tried something like that and the security tests failed since i used `constanize` on a string i received from a user. @mpovolny how do you handle these situations?